### PR TITLE
[calendar] setting up React and MicroFrontEnd for calendar project

### PR DESCRIPTION
**Pull request by behradhajmiri**
Please review. Thanks

### CODE REVIEW

**Review by 98H — APPROVED** [review]
Awesome! This Works Perfect.

File: apps/calendar/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+const { ModuleFederationPlugin } = require("webpack").container;
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+// const deps = require('./package.json').dependencies;
```
Comment (ketabi) [inline apps/calendar/webpack.config.js]: Please remove all commented codes.

File: apps/calendar/src/bootstrap.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
```
Comment (ketabi) [inline apps/calendar/src/bootstrap.js:1]: It would be better to import from zarkit

File: apps/calendar/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    new ModuleFederationPlugin({
+      name: 'calendarApp',
+      filename: 'remoteEntry.js',
+  
+      exposes: {
+        './RemoteApp': './src/Calendar',
+      },
+      
+      remotes: {
+        // "zarkit": "zarkit@http://localhost:3002/remoteEntry.js",
```
Comment (ketabi) [inline apps/calendar/webpack.config.js]: uncomment and fix the errors

**Review by ketabi — APPROVED** [review]

**Pull request merged**